release-content/0.14/release-notes/10164_Meshlet_rendering_initial_feature.md

```diff
@@ -1 +1,66 @@
-TODO
+After several months of hard work, we're super excited to bring you an experimental release of a new virtual geometry feature!
+
+This new rendering feature works much like Unreal Engine 5's Nanite renderer. You can take a very high-poly mesh, preprocess it to generate a MeshletMesh during build time, and then at runtime render huge amounts of geometry - much more than Bevy's standard renderer can support. No explicit LODs are needed - it's all automatic, and near seamless.
+
+This feature is still a WIP, and comes with several constraints, so be sure to read the docs and report any bugs you encounter. Look forward to more performance improvements (and associated breaking changes) in future releases.
+
+This feature does not use GPU "mesh shaders". Older GPUs are supported, but not recommended, and may become unsupported in a future release.
+
+In addition to the below user guide, checkout:
+* [The Bevy example for this feature](https://github.com/bevyengine/bevy/blob/release-0.14.0/examples/3d/meshlet.rs)
+* [The technical deep dive article by the main author of this feature](https://jms55.github.io/posts/2024-06-09-virtual-geometry-bevy-0-14)
+
+![Example scene using meshlets](meshlets.png)
```

Comment:
Maybe add a sentence saying what is shown in this screenshot :)

Reply:
```suggestion
![Two rows of colorful bunnies (hailing from Stanford) are lined up. On the right,. normal high-polygon geometry is shown. On the left, their meshes have been broken up into patchwork regions, each with a different color.](meshlets.png)
```